src/aiida/orm/groups.py

```diff
@@ -51,6 +52,28 @@ def load_group_class(type_string: str) -> Type['Group']:
     return group_class
 
 
+class GroupMeta(EntityFieldMeta):
```

Comment:
This class is raising issues w.r.t. the `_type_string` attribute. This might be due to changes done w.r.t. the attribute much later than Chris's initial work. However, the class goes unused. @chrisjsewell can you comment on your intentions here?

Reply:
This is just an error in the rebase. The `GroupMeta` used to be used, but the `_type_string` attribute was renamed to `__type_string` and is now defined by the `_type_string` class property. So you can delete this class

Reply:
I see. Discarding class.